decidim-proposals/app/cells/decidim/proposals/content_suggestion_section_cell.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Proposals
+    class ContentSuggestionSectionCell < Decidim::ViewModel
+      include Cell::ViewModel::Partial
+      include Decidim::TooltipHelper
+      include Decidim::CardHelper
+      include Decidim::LayoutHelper
+      include ApplicationHelper
+
+      def show
+        render :suggestions_card
+      end
+
+      def suggested_proposals
+        case suggested_content_criteria
+        when "most_recent"
+          recent_proposals
+        when "location"
+          proposals = proposals_by_location
+          proposals += random_proposals(suggested_content_limit - proposals.length) if proposals.length < suggested_content_limit
+          proposals
+        else
+          random_proposals(suggested_content_limit)
+        end
+      end
+
+      def base_query
+        Decidim::Proposals::Proposal.published.not_hidden.not_withdrawn.where(component: current_participatory_space.components.where(manifest_name: "proposals").published)
```

Comment:
Let's not promote rejected proposals :) 
```suggestion
        Decidim::Proposals::Proposal.published.except_rejected.not_hidden.not_withdrawn.where(component: current_participatory_space.components.where(manifest_name: "proposals").published)
```